cmd/src/batch_common.go

```diff
@@ -238,6 +243,14 @@ func batchExecute(ctx context.Context, out *output.Output, svc *batches.Service,
 				block.Write(repo.Name)
 			}
 			block.Close()
+		} else if repoSet, ok := err.(batches.IgnoredRepoSet); ok {
+			batchCompletePending(pending, "Resolved repositories")
+
+			block := out.Block(output.Line(" ", output.StyleWarning, "Some repositories contain .batchignore files and will be skipped. Use the -no-ignore flag to avoid skipping them."))
```

Comment:
Maybe a slight wording tweak, since this implies that you just get an opaque set of repositories, but then we print them:

```suggestion
			block := out.Block(output.Line(" ", output.StyleWarning, "The repositories listed below contain .batchignore files and will be skipped. Use the -no-ignore flag to avoid skipping them."))
```